cost/azure/unused_ip_addresses/azure_unused_ip_addresses.pt

```diff
@@ -549,6 +565,7 @@ EOS
         path "resourceId"
       end
     end
+    hash_exclude "ip_list"
```

Comment:
Would you be able to elaborate on what this does? It looks like "ip_list" is no longer one of the possible fields in the datasource we're validating, so it's not clear to me what this is doing.

Reply:
I forgot to remove that, acctually the good hash_exclude is a couple of lines above. Thanks You for addressing